.github/workflows/update_available_software.yml

```diff
@@ -34,7 +34,10 @@ jobs:
           cp docs/available_software/data/json_data.json docs/available_software/data/json_data.json.orig
           cp docs/available_software/data/json_data_detail.json docs/available_software/data/json_data_detail.json.orig
 
+          export TIME_GENERATED_TEMPLATE="{{ generated_time }}"
```

Comment:
Why the side-step here via this environment variable, why not just "hardcode" `{{ generated_time }}` in the generated pages?

Reply:
So that others sites can use the script without being forced to take the same approach.

Reply:
OK, doesn't hurt I guess.